lib/utils/new-module.js

```diff
@@ -166,16 +166,15 @@ function isDestructuring(node) {
 }
 
 function isIdentifierImportedFrom(node, module) {
```

Comment:
A test for this util function would be nice.

Reply:
I'm going to need this change in #552 as well -- I'll isolate the modification into its own commit and add some tests around the new behavior.

Reply:
I take that base, decided to write a separate method in #552 that leverages the `context` to get parent nodes, as documented.